src/app/ReadHandler.cpp

```diff
@@ -515,6 +515,17 @@ CHIP_ERROR ReadHandler::ProcessEventPaths(EventPathIBs::Parser & aEventPathsPars
         }
         ReturnErrorOnFailure(err);
 
+        err = path.GetIsUrgent(&(clusterInfo.mIsUrgentEvent));
+        if (CHIP_END_OF_TLV == err)
+        {
+            err = CHIP_NO_ERROR;
+        }
+        else if (err == CHIP_NO_ERROR)
+        {
+            VerifyOrReturnError(!clusterInfo.HasWildcardEventId(), err = CHIP_ERROR_IM_MALFORMED_EVENT_PATH);
```

Comment:
Why?  The spec allows a wildcard subscription to have IsUrgent set.

Test needed.

Reply:
I see, previously don't get the spec point that  the wildcard subscription to have IsUrgent set. Updated. Also added wildcard event with urgency flag. thanks